src/lib/util/Music/ManagerWrapper.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ *   Copyright 2019-2020 Jacz
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+import type { SkyraClient } from '@lib/SkyraClient';
+import { WSGuildCreate } from '@lib/types/DiscordAPI';
+import { DiscordPacket, LavalinkNodeOptions, Manager as LavacordManager, ManagerOptions } from 'lavacord';
+
+export class Manager extends LavacordManager {
+
+	public constructor(public readonly client: SkyraClient, nodes: LavalinkNodeOptions[], options?: ManagerOptions) {
+		super(nodes, options || {});
+		this.client = client;
+
+		this.send = packet => {
+			if (this.client.guilds.size) {
```

Comment:
duplicated code, if this is from @lavacord/discord.js then it would've been cause I support v11, v12 before managers and v12

Reply:
It's because @lavacord/klasa-core repo is as of yet an empty repository so yes I took the code of @lavacord/discord.js and put it in our file with proper license attribution so we have no dependency on whenever you will get around and actually make @lavacord/klasa-core